src/main/scala/org/allenai/wikitables/WikiTablesSemanticParserCli.scala

```diff
@@ -49,38 +49,43 @@ class WikiTablesSemanticParserCli extends AbstractCli() {
   
   var trainingDataOpt: OptionSpec[String] = null
   var testDataOpt: OptionSpec[String] = null
-  
+  val typeDeclaration = new WikiTablesTypeDeclaration()
```

Comment:
Declare this inside run() and pass it to the preprocessExamples method. That way we can swap in different type declarations if necessary.